src/main/java/org/synyx/urlaubsverwaltung/mail/Mail.java

```diff
@@ -0,0 +1,145 @@
+import java.util.Map;
+
+public class Mail {
+
+    private final List<Person> mailAddressRecipients;
+    private final MailNotification mailNotificationRecipients;
+    private final boolean sendToTechnicalMail;
+    private final boolean sendToEachIndividually;
+
+    private final String templateName;
+    private final Map<String, Object> templateModel;
+
+    private final String subjectMessageKey;
+    private final Object[] subjectMessageArguments;
+
+    private final List<MailAttachment> mailAttachments;
+
+    public Mail(List<Person> mailAddressRecipients, MailNotification mailNotificationRecipients, boolean sendToTechnicalMail,
+                boolean sendToEachIndividually, String templateName, Map<String, Object> templateModel, String subjectMessageKey,
+                Object[] subjectMessageArguments, List<MailAttachment> mailAttachments) {
+        this.mailAddressRecipients = mailAddressRecipients;
+        this.mailNotificationRecipients = mailNotificationRecipients;
+        this.sendToTechnicalMail = sendToTechnicalMail;
+        this.sendToEachIndividually = sendToEachIndividually;
+        this.templateName = templateName;
+        this.templateModel = templateModel;
+        this.subjectMessageKey = subjectMessageKey;
+        this.subjectMessageArguments = subjectMessageArguments;
+        this.mailAttachments = mailAttachments;
+    }
+
+    public List<Person> getMailAddressRecipients() {
+        return mailAddressRecipients;
+    }
+
+    public MailNotification getMailNotificationRecipients() {
+        return mailNotificationRecipients;
+    }
+
+    public boolean isSendToTechnicalMail() {
+        return sendToTechnicalMail;
+    }
+
+    public boolean isSendToEachIndividually() {
+        return sendToEachIndividually;
+    }
+
+    public String getTemplateName() {
+        return templateName;
+    }
+
+    public Map<String, Object> getTemplateModel() {
+        return templateModel;
+    }
+
+    public String getSubjectMessageKey() {
+        return subjectMessageKey;
+    }
+
+    public Object[] getSubjectMessageArguments() {
+        return subjectMessageArguments;
+    }
+
+    public List<MailAttachment> getMailAttachments() {
+        return mailAttachments;
+    }
+
+    public static Mail.Builder builder() {
+        return new Mail.Builder();
+    }
+
+    public static class Builder {
+
+        private final List<Person> mailAddressRecipients = new ArrayList<>();
+        private MailNotification mailNotificationRecipients;
+        private boolean sendToTechnicalMail;
+        private boolean sendToEachIndividually;
+
+        private String templateName;
+        private Map<String, Object> templateModel = new HashMap<>();
+
+        private String subjectMessageKey;
+        private Object[] subjectMessageArguments;
+
+        private final List<MailAttachment> mailAttachments = new ArrayList<>();
+
+        public Mail.Builder withRecipient(boolean sendToTechnicalMail) {
```

Comment:
Yea this is not that nice, but I thought we will get rid of the technical email after the google calendar deletion and therefore it was "ok" for me atm because I wanted to have "one" method to add recipients.

Reply:
on the other hand I thought that we maybe could use a "technical role" but that a "technical" needs to have a account.